metadata-ingestion/src/datahub/ingestion/source/vertexai.py

```diff
@@ -0,0 +1,715 @@
+                },
+                created=TimeStampClass(
+                    datetime_to_ts_millis(model_version.version_create_time)
+                )
+                if model_version.version_create_time
+                else None,
+                lastModified=TimeStampClass(
+                    datetime_to_ts_millis(model_version.version_update_time)
+                )
+                if model_version.version_update_time
+                else None,
+                version=VersionTagClass(versionTag=str(model_version.version_id)),
+                groups=[model_group_urn],  # link model version to model group
+                trainingJobs=[training_job_urn]
+                if training_job_urn
+                else None,  # link to training job
+                deployments=endpoint_urns,
+                externalUrl=self._make_model_version_external_url(model),
+                type="ML Model",
+            )
+        )
+
+        # TODO Add a container for Project as parent of the dataset
+        # aspects.append(
+        #     ContainerClass(
+        #             container=self._get_project_container().as_urn(),
+        #     )
+        # )
+
+        yield from MetadataChangeProposalWrapper.construct_many(
+            entityUrn=model_urn, aspects=aspects
+        )
+
+    def _search_endpoint(self, model: Model) -> List[Endpoint]:
+        """
+        Search for an endpoint associated with the model.
+        """
+        if self.endpoints is None:
+            endpoint_dict = defaultdict(list)
+            for endpoint in self.client.Endpoint.list():
+                for resource in endpoint.list_models():
+                    endpoint_dict[resource.model].append(endpoint)
+            self.endpoints = endpoint_dict
+
+        endpoints = self.endpoints[model.resource_name]
+        return endpoints
+
+    def _make_ml_model_urn(self, model_version: VersionInfo, model_name: str) -> str:
+        urn = builder.make_ml_model_urn(
+            platform=self.platform,
+            model_name=f"{model_name}_{model_version.version_id}",
+            env=self.config.env,
+        )
+        return urn
+
+    def _make_job_urn(self, job: VertexAiResourceNoun) -> str:
+        job_id = self._make_vertexai_job_name(entity_id=job.name)
+        urn = builder.make_data_process_instance_urn(dataProcessInstanceId=job_id)
+        return urn
+
+    def _make_vertexai_model_group_name(
+        self,
+        entity_id: str,
+    ) -> str:
+        separator: str = "."
+        return f"{self.config.project_id}{separator}model_group{separator}{entity_id}"
+
+    def _make_vertexai_endpoint_name(self, entity_id: str) -> str:
+        separator: str = "."
+        return f"{self.config.project_id}{separator}endpoint{separator}{entity_id}"
+
+    def _make_vertexai_model_name(self, entity_id: str) -> str:
+        separator: str = "."
+        return f"{self.config.project_id}{separator}model{separator}{entity_id}"
```

Comment:
same for all other methods - remove separator and update the param name

```suggestion
    def _make_vertexai_model_name(self, model_id: str) -> str:
        return f"{self.config.project_id}.model.{model_id}"
```

Reply:
done